andela/ah-alpha-frontend
# #163796801 add more rules for Linting

#### What does this PR do?
* Add more rules for linting

#### Description of Task to be completed?
* add more rules to our .eslintrc file
* add package.json-lock to .gitignore file

#### How should this be manually tested?
After cloning the repository
git checkout `ch-add-linting-rules-163796801`
- to see if linting is setup run `npm run  lint`
- if issues arise run `npm run lint-fix`
- all isues will be fixed automatically

#### What are the relevant pivotal tracker stories?

[#163796801](https://www.pivotaltracker.com/story/show/163796801)
